Backpack-SwiftUI/Skeleton/Classes/BPKSkeleton.swift

```diff
@@ -0,0 +1,99 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_Common
+
+public struct BPKSkeleton: View {
+    private var type: BPKSkeleton.ViewType = .image
+    private var size: BPKSkeleton.Size = .default
+    private var style: BPKSkeleton.Style = .default
```

Comment:
```suggestion
    private let type: BPKSkeleton.ViewType
    private let size: BPKSkeleton.Size
    private let style: BPKSkeleton.Style
```

We're setting these values in the initializer, there is no need to 'pre-set' them here. 
As a nice side-effect, we can now change them to let :D

Reply:
Perfect catch! Thank you! sorry for my un-pro swift!